### ISSUE TITLE
After Battle XP/Item/Gold screen text and border enlargement. (Possible brightness/shade difference)

### ISSUE THREAD

**Issue by Ehko4** [Legend-of-Dragoon-Modding/Severed-Chains-Public-Test#93]
**Describe the bug**
The text and border around SC after battle screen is quite bulbous when compared to retail. I previously mentioned it looked better on fade out, but I believe it's due to eliminating shading/lighting around the letters. There are slight differences in the shade of colors from one to the other.

**To Reproduce**
1. Go to any after battle victory.

https://github.com/Legend-of-Dragoon-Modding/Severed-Chains-Public-Test/assets/170750797/3a4374f4-4499-418c-b760-f4446ae94e8a

SC Left - Retail Duck Right

![Discord_4ErMyCTnQW](https://github.com/Legend-of-Dragoon-Modding/Severed-Chains-Public-Test/assets/170750797/3491a22a-9954-4bb7-8490-863c108379c6)

**Comment by Wulves** [comment]
Yeah I can see it on my end too. Looks like the border is thinner on the right hand side of each window in retail. SC's text looks **bold** by comparison, and maybe squished down a bit. I also think the background of the text box is making the letters appear even more bold, since Dart's EXP text looks less sharp than Shana's EXP text. However, that doesn't appear to be the case in retail.
